tests/cli/test_build_and_deploy_vmware.sh

```diff
@@ -81,6 +81,9 @@ version = "*"
 [[customizations.user]]
 name = "root"
 key = "$PUB_KEY"
+
+[customizations.kernel]
```

Comment:
Is this `[customizations.kernel]` or `[[customizations.kernel]]`, see above ?

Reply:
It is not a list, so it is [customizations.kernel]

The [[foo]] form is for things that can be listed multiple times, like users.